test/edu/psu/sweng500/team8/solver/HintGeneratorTests.java

```diff
@@ -11,7 +11,7 @@
 
 public class HintGeneratorTests {
 	@Test
-	public void canGetHintWhenACellHasOnlyOneAvailableNumber() {
+	public void cangetHintWhenACellHasOnlyOneAvailableNumber() {
```

Comment:
I must have been updated when I did a mass update for unconventional method names starting with capital G. I will update them back.